version.config

```diff
@@ -1,2 +1,2 @@
-version = 2.3.0-dev
-update_from_version = 2.2.0
+version = 2.4.0-dev
```

Comment:
I don't think we are doing this bump for a patch release, right? We are still on 2.3.0-dev since we're releasing 2.2.1.

Reply:
Fixed.